src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] testFigure = new Figure[6];
+        testFigure[0] = figureSupplier.getRandomFigure();
+        testFigure[1] = figureSupplier.getRandomFigure();
+        testFigure[2] = figureSupplier.getRandomFigure();
+        testFigure[3] = figureSupplier.getDefaultFigure();
+        testFigure[4] = figureSupplier.getDefaultFigure();
+        testFigure[5] = figureSupplier.getDefaultFigure();
```

Comment:
It would be better to add figures in the `loop`
You can create `for` loop and then check if it is the first half - then add RandomFigure or second half - add DefaultFigure